src/components/widgets/BaseInput.js

```diff
@@ -18,6 +19,7 @@ function BaseInput(props) {
       {...inputProps}
       className="form-control"
       readOnly={readonly}
+      autoFocus={autoFocus}
```

Comment:
@n1k0 I am curious about the reason why we need lowercasing html atrribute names, could you help to explain